src/web/index.html

```diff
@@ -111,11 +110,12 @@
     </script>
     <style>
         html, body {
+            display:flex;
```

Comment:
Is there any benefit in using flex?

Reply:
All the modern style uses flexbox, for this reason it is required that the body and the html have this tag for correct operation, the flex display does not affect the other styles